frontend/projects/marketplace/src/types.ts

```diff
@@ -30,6 +30,7 @@ export interface MarketplacePkg {
     [id: string]: {
       title: string
       icon: Url
+      'is-local': boolean
```

Comment:
:question: Is this guaranteed to exist?

Reply:
:pleading_face: Could I get any link that any other service is changing to use this.

The only thing that I have found to have this field is the [api fixures ](https://github.com/Start9Labs/embassy-os/pull/2110/files#diff-c96a082192e9e6068d0e16576e235a91fd429993f8d3f203fc64c6c04374c1d8)
Guessing that the service changes are needed, would like the link after changes.

Reply:
It will come from the registry